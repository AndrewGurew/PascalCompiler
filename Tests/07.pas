program test;
var
a: Integer;
		b:String;








c:Integer;





									d:String; 