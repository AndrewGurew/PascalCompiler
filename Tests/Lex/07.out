Position            Type                     Text                Value               
-------------------------------------------------------------------------------------
(1,1)               Programm                 program             program             
(1,9)               Id                       test                test                
(1,13)              Semicolon                ;                   ;                   
(2,2)               Var                      var                 var                 
(3,2)               Id                       a                   a                   
(3,3)               Colon                    :                   :                   
(3,5)               Integer                  Integer             Integer             
(3,12)              Semicolon                ;                   ;                   
(4,4)               Id                       b                   b                   
(4,5)               Colon                    :                   :                   
(4,6)               Id                       String              String              
(4,12)              Semicolon                ;                   ;                   
(13,2)              Id                       c                   c                   
(13,3)              Colon                    :                   :                   
(13,4)              Integer                  Integer             Integer             
(13,11)             Semicolon                ;                   ;                   
(19,11)             Id                       d                   d                   
(19,12)             Colon                    :                   :                   
(19,13)             Id                       String              String              
(19,19)             Semicolon                ;                   ;                   
(19,19)             End of file              EndOfFile           EndOfFile           
